### PR TITLE
ci: Include PR url in msg body

### PR DESCRIPTION
**Pull request by MarioCarrilloA**
This commmit adds the url of the pull request
in the message body of the report.

Fixes: #508

Signed-off-by: Mario Alfredo Carrillo Arevalo <mario.alfredo.c.arevalo@intel.com>

### CODE REVIEW

**Comment by MarioCarrilloA** [comment]
This commit should be merged until https://github.com/clearcontainers/tests/pull/507 is merged first

File: .ci/run_metrics_ci.sh [hunk 1]
```diff
@@ -87,9 +88,11 @@ pushd "$CURRENTDIR/../metrics"
 	bash storage/fio_job.sh -b 16k -o read -t "storage IO linear read bs 16k"
 	bash storage/fio_job.sh -b 16k -o write -t "storage IO linear write bs 16k"
 
+	# Pull request URL
+	PR="$GITHUB_URL/$LOCALCI_REPO_SLUG/pull/$LOCALCI_PR_NUMBER"
 
 	# Parse/Report results
-	emailreport
+	emailreport -c "Pull request: $PR"
```
Comment (grahamwhaley) [inline .ci/run_metrics_ci.sh]: just a note - would be nice if we could get $LOCALCI_PR_NUMBER into the email subject line. Actually, probably more like:
`$LOCALCI_REPO_SLUG:$LOCALCI_PR_NUMBER` or similar. hmm, second thoughts - maybe 'github style' ?
`[$LOCAL_CI_REPO_SLUG] <report subject goes here> (#$LOCALCI_PR_NUMBER)`

thoughts?

Comment (MarioCarrilloA) [reply .ci/run_metrics_ci.sh]: ok, I can add "-s" option in order to overwrite the subject in the email

**Comment by chavafg** [comment]
lgtm

**Pull request merged**